reference/webhooks.v3.yml

```diff
@@ -1,463 +1,642 @@
-                destination: 'https://665b65a6.ngrok.io/webhooks`'
-                is_active: true
-                headers:
-                  custom: string
-  securitySchemes:
-    X-Auth-Token:
-      type: apiKey
-      name: X-Auth-Token
-      in: header
-      description: |-
-        ### OAuth Scopes
-        |  **UI Name** | **Permission** | **Parameter** |
-        | --- | --- | --- |
-        |  - | - | - |
-
-        **Note**: Creating and mangaing webhooks require the default scope used when an API User account is created.
-
-        ### Headers
-
-        |Header|Parameter|Description|
-        |-|-|-|
-        |`X-Auth-Token`|`access_token `|Obtained by creating an API account or installing an app in a BigCommerce control panel.|
-
-        ### Example
-
-        ```http
-        GET /stores/{$$.env.store_hash}/v3/catalog/summary
-        host: api.bigcommerce.com
-        Content-Type: application/json
-        X-Auth-Token: {access_token}
-        ```
-
-        * For more information on Authenticating BigCommerce APIs, see: [Authentication](https://developer.bigcommerce.com/api-docs/getting-started/authentication).
   schemas:
-    error_Full:
+    Webhook:
       type: object
-      title: error_Full
       properties:
-        status:
-          description: |
-            The HTTP status code.
+        id:
+          description: Unique ID of Webhook.
           type: integer
-        title:
-          description: |
-            The error title describing the particular error.
+          format: int32
+        client_id:
+          description: 'API User Client ID, unique to the store.'
           type: string
-        type:
+        store_hash:
+          description: Unique Hash of the store.
           type: string
-    errorDetailed_Full:
-      type: object
-      title: errorDetailed_Full
-      properties:
-        errors:
+        scope:
+          description: The event that the webhook is subscribed to.
+          type: string
+          example: store/order/*
+        destination:
+          description: 'URL to which the webhook is sent. URL must be active, return a 200 response, and be served on port 443 (custom ports not supported).'
+          type: string
+          format: url
+        headers:
+          description: Custom headers to be included in destination url request.
           type: object
           additionalProperties:
             type: string
-          title: DetailedErrors
-    webhook_Base:
+          properties:
+            custom:
+              type: string
+        is_active:
+          description: Identifies whether webhook is active.
+          type: boolean
+        created_at:
+          description: UTC timestamp in seconds that the webhook was created.
+          type: integer
+          format: int64
+        updated_at:
+          description: UTC timestamp in seconds that the webhook was last updated.
+          type: integer
+          format: int64
+    WebhookCreate:

```

Comment:
I see you are changing the naming convention. Is there a reason for this update?